include/revng/Model/Register.h

```diff
@@ -114,10 +165,53 @@ type: enum
   - name: s5_mips
   - name: s6_mips
   - name: s7_mips
+  - name: s8_mips
+  - name: t0_mips
+  - name: t1_mips
+  - name: t2_mips
+  - name: t3_mips
+  - name: t4_mips
+  - name: t5_mips
+  - name: t6_mips
+  - name: t7_mips
+  - name: t8_mips
+  - name: t9_mips
```

Comment:
* How did we miss `t0`, `t1` registers? Weird
* There's no `s8`

Reply:
ABI doc mentions `s8` as an alias for `$30` memory "cell" and it's also a callee-preserved register.

Reply:
Now that you mention it, the register list from the ABI doc doesn't mention `fp`! So that's how it is, I'm sorry, I should have been more careful with that.

Reply:
Gone.